Dockerfile

```diff
@@ -32,6 +32,8 @@ RUN NODE_ENV=production npm run build && \
 # development image
 # docker build . --target=development
 FROM node:$VERSION-alpine as development
+# https://github.com/nodejs/docker-node/blob/main/docs/BestPractices.md#handling-kernel-signals
+RUN apk add --no-cache tini
```

Comment:
maybe we can run this once and create a "wrapped" build stage to use as the base for development and production targets

Reply:
updated in fe13e5a